### PR TITLE
Fix write compression.

### PR DESCRIPTION
**Pull request by rubensf**
On the midst of PR chains and rebasing, I managed to
leave the functionality off.

### CODE REVIEW

File: go/pkg/client/bytestream.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ func (c *Client) WriteBytes(ctx context.Context, name string, data []byte) error
 
 // WriteChunked uploads chunked data with a given resource name to the CAS.
 func (c *Client) WriteChunked(ctx context.Context, name string, ch *chunker.Chunker) error {
+	fmt.Println("name: ", name)
```
Comment (ola-rozenfeld) [inline go/pkg/client/bytestream.go:29]: debug printf

**Review by ola-rozenfeld — COMMENTED** [review]
But how did the tests pass, then?

File: go/pkg/client/bytestream.go (hunk 1 above)
Comment (rubensf) [reply go/pkg/client/bytestream.go]: Done - thanks for catching!

**Review by rubensf — COMMENTED** [review]
The fake cas server is meant to reply with the appropriate blob type (compressed or uncompressed). The issue was that compressed writes were never being done.

Now the Write should be doing compressed writes when necessary, as confirmed by the debug print I just deleted :)

I arguably should expand the fake cas server to signal that it should expect a request to come compressed or uncompressed.

**Review by ola-rozenfeld — APPROVED** [review]

**Pull request merged**